.nvmrc

```diff
@@ -1 +1 @@
-10.0.0
+12.9.0
```

Comment:
a small nit for next time - so we're on the same page - i like to adhere to the guidance that production apps should use `ACTIVE`, not `CURRENT` according to this schedule. In this case, we are talking a few days, but it's wanted to bring attention to it.

Reply:
Totally fair — happy to bump to down to 11.x (but yep, 9 days to go till LTS!). 

LMK if you think we should change this before merging down.